docs/asciidoc/modules/ROOT/pages/database-integration/vectordb/custom.adoc

```diff
@@ -0,0 +1,96 @@
+
+== Custom (i.e. other vector databases)
+
+Here is a list of all available Qdrant procedures:
```

Comment:
typo Qdrant?

Reply:
Perhaps a bit more introduction on the generic vectordb procedures why/how/what?